icu4c/source/common/unicode/utypes.h

```diff
@@ -384,6 +384,81 @@ typedef double UDate;
 #define U_TOOLUTIL_API U_IMPORT
 #endif
 
+/**
+ * \def U_DATA_API_CLASS
+ * Set to export library symbols from inside the stubdata library,
+ * and to import them from outside, to be used on a class.
+ * @draft ICU 78
```

Comment:
Please also send a [proposal email](https://icu.unicode.org/processes/proposal-template) for the new definitions.

Reply:
Done: https://groups.google.com/a/unicode.org/g/icu-design/c/-Q82BiedfTQ